squad/frontend/group_settings.py

```diff
@@ -30,6 +31,28 @@ def settings(request, group_slug):
     return render(request, 'squad/group_settings/index.jinja2', context)
 
 
+class NewProjectForm(forms.ModelForm):
+    class Meta:
+        model = Project
+        fields = ['slug', 'name', 'is_public', 'description']
+
+
+@auth_write
+def new_project(request, group_slug):
+    group = request.group
+    if request.method == "POST":
+        form = NewProjectForm(request.POST, instance=Project(group=group))
+        if form.is_valid():
+            project = form.save()
+            return redirect(reverse('project-settings', args=[group.slug, project.slug]))
```

Comment:
I think exceptions are not handled properly here. For example attempt to create project with slug that already exists in database results in ugly exception rather than error message in the UI.